python/lsst/ts/Dome/dome_csc.py

```diff
@@ -213,14 +238,14 @@ async def do_moveAz(self, data):
             Contains the data as defined in the SAL XML file.
         """
         self.assert_enabled()
-        self.log.info(
-            f"Moving Dome to azimuth {data.azimuth} and gthen start crawling at azRate {data.azRate}"
+        self.log.debug(
+            f"Moving Dome to azimuth {data.position} and then start crawling at azRate {data.velocity}"
         )
         await self.write_then_read_reply(
             command="moveAz",
             parameters={
-                "azimuth": math.radians(data.azimuth),
-                "azRate": math.radians(data.azRate),
+                "position": math.radians(data.position),
+                "velocity": math.radians(data.velocity),
```

Comment:
I strongly recommend having `write_then_read_reply` take command and ``**kwargs`` as parameters. That  clarifies the API, makes it easier to call correctly and harder to call incorrectly. For example:
```
    await self.write_then_read_reply("moveEl", position=math.radians(data.position))
    # or, if you prefer (both work fine):
    await self.write_then_read_reply(command="moveEl", position=math.radians(data.position))
```
Where the method is implemented as something like:
```
async def write_then_read_reply(self, command, **params):
    command_dict = dict(command=command, parameters=params)
    command_encoded = encoding_tools.encode(**command_dict)
```